02 - Community Expansions/02.05 All Community Expansions/Plugins/Straightforward-Obsidian-to-Latex.md

```diff
@@ -0,0 +1,11 @@
+Github Repo: https://github.com/MariosGkMeng/Straightforward-Obsidian2Latex
+
+
+%%
+[[LaTeX]]
+%%
+
+
+# This note in GitHub
```

Comment:
Is this plugin in the Community Store?

The reason I ask is that:

1. Once plugins are approved in the community store, some automation in this project means that the new plugin page is created automatically.
2. There is a template to use to create plugin pages, and this new file does not use it, so does not contain all the standard information we would normally expect to see.

Reply:
Reading the Contributing section in this folder, I do read it as saying that this folder is for plugins in the obsidian-releases repository, in other words, plugins that have been reviewed and approved by the Obsidian team:

https://publish.obsidian.md/hub/02+-+Community+Expansions/02.05+All+Community+Expansions/Plugins/🗂%EF%B8%8F+Plugins#Contributing

Reply:
https://github.com/MariosGkMeng/Straightforward-Obsidian2Latex is not a plugin.